tools/Customization/DevHome.Customization/Views/DevDriveInsightsView.xaml

```diff
@@ -0,0 +1,453 @@
+                </StackPanel>
+            </DataTemplate>
+
+            <!-- Listview Template for showing the Dev Drive cards within a DevDrivesListViewModel object. -->
+            <DataTemplate x:Key="DevDriveLoadedTemplate" x:DataType="devDriveModels:DevDrivesListViewModel">
+                <ListView
+                    x:Name="DevDrivesList"
+                    AutomationProperties.Name="{Binding AccessibilityName, Mode=OneWay}"
+                    HeaderTemplate="{StaticResource DevDriveContainerHeaderTemplate}"
+                    DataContext="{Binding}"
+                    ItemsSource="{x:Bind DevDriveCardAdvancedCollectionView, Mode=OneWay}"
+                    SelectionMode="Single"
+                    ItemTemplate="{StaticResource HorizontalCardForDevDriveInsightsPage}"
+                    Margin="0 0 0 10"
+                    ItemContainerStyle="{StaticResource HorizontalCardListViewItemContainerStyle}"
+                    IsMultiSelectCheckBoxEnabled="False"
+                    SelectedItem="{x:Bind SelectedItem, Mode=OneWay}">
+                    <i:Interaction.Behaviors>
+                        <ic:EventTriggerBehavior EventName="SelectionChanged">
+                            <ic:InvokeCommandAction  
+                                Command="{Binding ContainerSelectionChangedCommand}"  
+                                CommandParameter="{Binding SelectedItem, ElementName=DevDrivesList}"/>
+                        </ic:EventTriggerBehavior>
+                    </i:Interaction.Behaviors>
+                </ListView>
+            </DataTemplate>
+
+            <!-- Listview Template for showing the Dev Drive optimizer cards within a DevDriveOptimizersListViewModel object. -->
+            <DataTemplate x:Key="DevDriveOptimizersLoadedTemplate" x:DataType="devDriveModels:DevDriveOptimizersListViewModel">
+                <ListView
+                    x:Name="DevDriveOptimizersList"
+                    AutomationProperties.Name="{Binding AccessibilityName, Mode=OneWay}"
+                    HeaderTemplate="{StaticResource DevDriveContainerHeaderTemplate}"
+                    DataContext="{Binding}"
+                    ItemsSource="{x:Bind DevDriveOptimizerCardAdvancedCollectionView, Mode=OneWay}"
+                    SelectionMode="Single"
+                    ItemTemplate="{StaticResource HorizontalCardForDevDriveOptimizerInsightsPage}"
+                    Margin="0 0 0 10"
+                    ItemContainerStyle="{StaticResource HorizontalCardListViewItemContainerStyle}"
+                    IsMultiSelectCheckBoxEnabled="False"
+                    SelectedItem="{x:Bind SelectedItem, Mode=OneWay}">
+                                <i:Interaction.Behaviors>
+                                    <ic:EventTriggerBehavior EventName="SelectionChanged">
+                                        <ic:InvokeCommandAction  
+                    Command="{Binding ContainerSelectionChangedCommand}"  

```

Comment:
Looks like the spacing on these lines isn't correct.

Reply:
Fixed.